May This Month in Solid Newsletter (#191)
solid/solidproject.org: _posts/newsletter/2020-05-07-this-month-in-solid.md

---
layout: newsletter
title: This Month in Solid 2020-05-07
permalink: /newsletter/2020-05-07
tags: [monthly, updates]
categories: [Updates]
---
{% include twis-intro.md %}

## Updates

### [Events](https://solidproject.org/events)

Solid World is an opportunity to meet people who are interested in or currently working on Solid.

A big thank you to Christian Buggedei, Walter Almeida, Glen Simister, Manoharan Ramachandran, and Michiel de Jong for presenting at Solid World May last Thursday.

> In the first talk, Christian Buggedei presented Darcy, a social network that knows nothing about you.Of note, Darcy recently won a Mozilla grant. You can find the Darcy source code here. Christian Buggedei is a founder and technology expert.

Reviewer: ```suggestion
In the first talk, Christian Buggedei presented Darcy, a social network that knows nothing about you. Of note, Darcy recently won a Mozilla grant. You can find the Darcy source code here. Christian Buggedei is a founder and technology expert.
```

Reviewer 2: ```suggestion
In the first talk, Christian Buggedei presented Darcy, a social network that knows nothing about you. Of note, Darcy recently won a Mozilla grant. You can find [the Darcy source code on GitHub](https://github.com/Darcy-Social/ibex). Christian Buggedei is a founder and technology expert.
```

> In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and Sparql endpoints using a location tracking Solid app that leverages dbPedia.org as a data source for countries and places. You can find the Go location app source code here.

Reviewer: *DBpedia (or just dbpedia)

Reviewer 2: ```suggestion
In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and Sparql endpoints using a location tracking Solid app that leverages DBpedia as a data source for countries and places. You can find the Go location app source code here.
```

Reviewer: missing link still

Reviewer 2: Right, updated suggestion:

```suggestion
In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and Sparql endpoints using a location tracking Solid app that leverages DBpedia as a data source for countries and places. You can find the [Go location app source code on GitHub](https://github.com/walteralmeida/GOForSolidVisitedPlacesApplication/).
```

> In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and Sparql endpoints using a location tracking Solid app that leverages DBpedia as a data source for countries and places. You can find the [Go location app source code on GitHub](https://github.com/walteralmeida/GOForSolidVisitedPlacesApplication/).

Reviewer: ```suggestion
In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and SPARQL endpoints using a location tracking Solid app that leverages DBpedia as a data source for countries and places. You can find the [Go location app source code on GitHub](https://github.com/walteralmeida/GOForSolidVisitedPlacesApplication/).
```

In the third talk, Manoharan Ramachandran presented the Solid app built by Open University's blockchain team, looking into how to support immunity passports as mentioned by the UK government daily briefings. The Solid app facilitates the instant verification of tamper-proof test results whilst preserving privacy.

> In the fourth talk, Glen Simister presented the DVO Profile Manager, a public profile (or resume), which anyone can create for free, and update. DVO Profile Manager works in conjunction with the DVO Toolbar, a browser extension which allows people to like, dislike, and comment on any URL. You can find the DVO Profile Manager source code here.

Reviewer: missing link

Reviewer 2: ```suggestion
In the fourth talk, Glen Simister presented the DVO Profile Manager, a public profile (or resume), which anyone can create for free, and update. DVO Profile Manager works in conjunction with the DVO Toolbar, a browser extension which allows people to like, dislike, and comment on any URL. You can find [the DVO Profile Manager source code on GitHub](https://github.com/dvo/profile).
```

> In the fifth talk, Michiel de Jong presented his latest experiment, SNAP on Solid, explaining how Solid inboxes can be used as the messaging layer for Network Ledger Technology. Check out the SNAP source code here.

Reviewer: missing link

Reviewer 2: ```suggestion
In the fifth talk, Michiel de Jong presented his latest experiment, SNAP on Solid, explaining how Solid inboxes can be used as the messaging layer for Network Ledger Technology. Check out the [SNAP source code on GitHub](https://github.com/ledgerloops/snap-solid/).
```

> If you missed Solid World you can [watch the recording [here](https://vimeo.com/415976385).

Reviewer 2: There's a duplicate `[` here:

```suggestion
If you missed Solid World you can [watch the recording on Vimeo](https://vimeo.com/415976385).
```

> The [next Solid World](https://www.eventbrite.com/e/solid-world-june-tickets-104631158612) will happen soon on the 4th June  2020. If you have any content requests or would like to present at the next Solid world email m.laszlo@inrupt.com.

Reviewer 2: Remove duplicate space, add link:

```suggestion
The [next Solid World](https://www.eventbrite.com/e/solid-world-june-tickets-104631158612) will happen soon on the 4th June 2020. If you have any content requests or would like to present at the next Solid world email [m.laszlo@inrupt.com](mailto:m.laszlo@inrupt.com).
```

### Articles mentioning Solid

* 2020-05-06 [What Happens When You Get Sick Right Now?](https://hackernoon.com/what-happens-when-you-get-sick-right-now-mh12x3y82)
* 2020-04-29 [New App for COVID-19 Combines Blockchain With Web Inventor's Privacy Tech](https://cointelegraph.com/news/new-app-for-covid-19-combines-blockchain-with-web-inventors-privacy-tech)
* 2020-04-15 [COVID-19 Antibody Test Certification: There's an app for that](https://arxiv.org/abs/2004.07376)
* 2020-04-14 [Scale, distribution, and occupying Mars: COVID-19 is a catalyst](https://www.zdnet.com/article/scale-distribution-and-occupying-mars-covid-19-is-a-catalyst/)

### In Other News

#### Solid Implementations

* [Solid Health](https://github.com/jasonpaulos/solid-health)
* [DVO](https://dvo1.github.io/profile/index.html)
* [Snap](https://michiels-nlt-kit.herokuapp.com)
* [Darcy](https://ibex.darcy.is/feed)

#### External Grants

> There are an EU initiative [DAPSI](https://dapsi.ngi.eu/) and [NGI Pointer](https://www.ngi.eu/ngi-projects/ngi-pointer/) that could be suitable for Solid app projects.

Reviewer: ```suggestion
There are EU initiatives [DAPSI](https://dapsi.ngi.eu/) and [NGI Pointer](https://www.ngi.eu/ngi-projects/ngi-pointer/) that could be suitable for Solid app projects.
```

The editorial team has been holding productive meetings every 2-3 weeks aimed at advancing progress towards normative Solid ecosystem specifications. Minutes are posted in the [Solid W3C Community Group Wiki](https://www.w3.org/community/solid/wiki/Meetings), including minutes from our meetings in April:
* [Minutes](https://www.w3.org/community/solid/wiki/Meetings#2020-04-08T13:30:00Z) from April 8th
* [Minutes](https://www.w3.org/community/solid/wiki/Meetings#2020-04-24_1600CET) from April 24th

Other notable activity related to the Solid specification include:

* A draft candidate proposal for WebID-OIDC is underway in the [Authentication Panel](https://github.com/solid/authentication-panel) based on rough consensus agreement around key elements of the protocol needed to support the development of reference implementations.
* Use cases for server managed auxiliary resources related to resource provenance, auditing, and access history are being undertaken in the [Data Interoperability Panel](https://github.com/solid/data-interoperability-panel).

> * Work towards a normative specification for [web access control](https://github.com/solid/web-access-control-spec) will begin soon, organized through the [Authorization Panel](https://github.com/solid/authorization-and-access-control-panel).

Reviewer: ```suggestion
* Work towards a normative specification for [Web Access Control](https://github.com/solid/web-access-control-spec) will begin soon, organized through the [Authorization Panel](https://github.com/solid/authorization-and-access-control-panel).
```
